### PR TITLE
Update review-logs-using-azure-monitor.md

### PR DESCRIPTION
**Pull request by msbemba**
AddedLog Analytics Contributor role needed to  Configure the collection of logs from Azure Storage. Reference  -https://docs.microsoft.com/en-us/azure/azure-monitor/logs/manage-access

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 please copy review

**Comment by PRMerger4** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/memdocs/issues/2297

File: memdocs/intune/fundamentals/review-logs-using-azure-monitor.md [hunk 1]
```diff
@@ -57,6 +57,7 @@ To use this feature, you need:
 * An Azure subscription that you can sign in to. If you don't have an Azure subscription, you can [sign up for a free trial](https://azure.microsoft.com/free/).
 * A Microsoft Intune environment (tenant) in Azure
 * A user who's a **Global Administrator** or **Intune Service Administrator** for the Intune tenant.
+* A user needs Log Analytics Contributor role in  Log Analytics workspaces to be able to Configure the collection of logs from Azure Storage.
```
Comment (JohanFreelancer9) [inline memdocs/intune/fundamentals/review-logs-using-azure-monitor.md:60]:
```suggestion
* A user needs a Log Analytics Contributor role in Log Analytics Workspace to be able to configure the collection of logs from Azure Storage.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by PRMerger6** [comment]
@MandiOhlinger : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by MandiOhlinger — APPROVED** [review]

**Comment by MandiOhlinger** [comment]
#sign-off

**Pull request merged**